robotoff/logos.py

```diff
@@ -327,10 +331,20 @@ def predict_logo_predictions(
 def generate_prediction(
     logo_type: str,
     logo_value: Optional[str],
+    data: Dict,
     automatic_processing: Optional[bool] = False,
-    **kwargs,
 ) -> Optional[Prediction]:
-    if logo_type not in LOGO_TYPE_MAPPING:
+    """Generate a Prediction from a logo.
+
+    The Prediction may either be created after the annotation of the logo by
+    a human (in which case `automatic_processing` is True), or by infering the
+    logo value from nearest neighbor labels (in which case
+    `automatic_processing` is False).
+
+    Currently, only brand and label logo types are supported: None is returned
+    if the logo type is different, or if the logo_value is None.
```

Comment:
This is true regarding automatic processing but not regarding human annotations isn't it ? I thought we could annotate also packager codes for example.

Reply:
Yes we can annotate logos of other type than label and brand, but only for these two types a prediction and insight is generated.